config/crd/bases/dynatrace.com_dynakubes.yaml

```diff
@@ -5005,6 +5005,70 @@ spec:
                 type: boolean
               extensions:
                 properties:
+                  otlpexporter:
+                    properties:
```

Comment:
in ticket it's written:
```
kind: DynaKube
spec:
  otelOtlpExporter:
    enableTraces: true|false
    enableMetrics: true|false
```

but you implemented in under extension section:
```
 spec:
   extensions:
     otlpexporter:
       enableLogs: true
       enableTraces: true
       enableMetrics: true
```

Reply:
BTW I saw your comment (others can easily miss it), but or you should change ticket description and validate it with PO or change implementation.

Reply:
fixed in ffd683a7711cb7a4c9de6641f49bad065b810287 as specified by PA.